src/modules/27-platform/delegates/pages/delegates/__tests__/DelegateListing.test.tsx

```diff
@@ -317,6 +320,9 @@ describe('Test delegate buttons', () => {
       fireEvent.click(filterBtn!)
     })
 
-    expect(container).toMatchSnapshot()
+    // Ensure that "Filter" drawer opens
+    await waitFor(() => {
+      expect(screen.getByText('platform.delegates.delegateIdentifier')).toBeInTheDocument()
+    })
```

Comment:
You can simplify `waitFor`/`getBy` combos by using `findBy` instead.
```suggestion
    expect(await screen.findByText('platform.delegates.delegateIdentifier')).toBeInTheDocument()
```

Does the exact same thing, so absolutely not a blocker.

Reply:
Thanks! I'll keep that in mind from next time onwards.